_posts/2015-06-30-GitHub-Integration.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: GitHub Integration
+status: public
+type: post
+published: true
+author: Go Team
+---
+
+In this blog we will see how you can seamlessly integrate Go CD with GitHub.
+
+### GitHub Login with [GitHub OAuth Plugin](https://github.com/srinivasupadhya/gocd-oauth-login)
+
+Login to Go CD through GitHub.
+
+* Configure plugin
+
+    <figure>
+      <img src="/images/blog/authentication-plugins/configure-plugin.png" class="has_border full_size"
+        alt="Figure 2: GoCD - Configure Plugin" id="mature_ci_cd_setup" title="GoCD - Configure Plugin" />
+      <figcaption>Figure 1: Configure plugin pop-up <span class="click_to_enlarge">(Click to enlarge)</span></figcaption>
+    </figure>
+
+* Ability to Search & Add users
```

Comment:
Would you mind stepping back a bit and describing where to get the OAuth keys?

Also can we call them "oauth client ID" and "oauth client secret". Would you prefer that I log a bug against https://github.com/srinivasupadhya/gocd-oauth-login?